### ISSUE TITLE
@extends not working with @interface

### ISSUE THREAD

**Issue by myphysicslab** [jsdoc3/jsdoc#897]
When interface B extends interface A, the documentation of interface A is lost.  By "lost" I mean the interface A methods do not appear in documentation of interface B.  And if class C implements interface B, then also methods defined in interface A do not have their documentation (they do appear, but with no documentation).

Below is an example where SubjectEvent extends Printable and GenericEvent implements SubjectEvent.  Note that in the GenericEvent documentation, the toStringShort method has no documentation.  This is available at
http://www.myphysicslab.com/rmnp/test20150129/
The resulting HTML file is at
http://www.myphysicslab.com/rmnp/test20150129/out/index.html

```javascript

/** @classdesc An object that has a minimal string representation via
the `toStringShort` method.
* @interface
*/
myphysicslab.lab.util.Printable = function() {};

/** Returns a minimal string representation of this object, usually
giving just identity information like the class name and name of the
object.
@method
@return {string} a minimal string representation of this object.
*/
myphysicslab.lab.util.Printable.prototype.toStringShort;

/** @classdesc Provides information about an event that has happened to
a {Subject}.
* @interface
* @extends {myphysicslab.lab.util.Printable}
*/
myphysicslab.lab.util.SubjectEvent = function() {};

/** Returns the name of this {SubjectEvent}.
@method
@return {string} the name of this {SubjectEvent}.
*/
myphysicslab.lab.util.SubjectEvent.prototype.getName;

/** @classdesc A simple implementation of a {SubjectEvent}, represents
some event that has occurred in a {Subject}.
@param {string} name the name of this event
@constructor
@implements {myphysicslab.lab.util.SubjectEvent}
*/
myphysicslab.lab.util.GenericEvent = function(name) {
  /**
  @type {string}
  @private
  */
  this.name_ = name;
};

/** @inheritDoc */
myphysicslab.lab.util.GenericEvent.prototype.getName = function() {
  return this.name_;
};

/** @inheritDoc */
myphysicslab.lab.util.GenericEvent.prototype.toStringShort = function() {
  return "GenericEvent{\""+this.name_+"\"}";
};
```

**Comment by hegemonic** [comment]
Confirmed. Thanks for reporting this issue. I'll investigate.

Further reduced testcase:

```js
'use strict';

/** @interface */
function IFoo() {}

/** IFoo myMethod */
IFoo.prototype.myMethod = function() {};

/**
 * @interface
 * @extends {IFoo}
 */
function IBar() {}

/**
 * @implements IBar
 */
function Bar() {}

/** @inheritdoc */
Bar.prototype.myMethod = function() {};
```

**Comment by hegemonic** [comment]
Fixed on master and releases/3.3.

**Comment by myphysicslab** [comment]
Wow, that was fast!  Thanks Jeff, it's working great now.  I'll keep
pounding away at it.
--ErikN
